src/main/java/seedu/address/model/Duration.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.model;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+/**
+ * A Duration contains a start time and an end time.
+ */
+public class Duration implements Comparable<Duration> {
+
+    private static final DateTimeFormatter TIME_FORMAT = DateTimeFormatter.ofPattern("HHmm");
```

Comment:
can have an additional time format other than TIME_FORMAT_FOR_INPUT which is TIME_FORMAT_FOR_PRINT for toString() if needed.

Reply:
Will do. Thanks.